### ISSUE TITLE
User startup.py is being executed TWICE

### ISSUE THREAD

**Issue by alankilborn** [bruderstein/PythonScript#339]
In version 3.0.18, add following code to user `startup.py`:

```py
try:
    hello
except:
    hello = 0
hello += 1
print('hello:', hello)
```

Make sure that the PythonScript console window is being shown when exiting Notepad++.

Upon restarting Notepad++, the code above will show that `startup.py` is being executed two times:

![image](https://github.com/bruderstein/PythonScript/assets/77065706/f9fb1f4d-f412-4079-a3a1-ce69caa82a1f)

---

Notepad++ Debug Info:

Notepad++ v8.6.8   (64-bit)
Build time : Jun  4 2024 - 00:30:00
Path : C:\NotepadPlusPlus\Misc\_NoBackup\Releases\npp.8.6.8\npp.8.6.8.portable.x64\notepad++.exe
Command Line : -titleAdd="8.6.8 64-bit" -multiInst
Admin mode : OFF
Local Conf mode : ON
Cloud Config : OFF
Periodic Backup : ON
OS Name : Windows 11 Pro (64-bit)
OS Version : 23H2
OS Build : 22631.3593
Current ANSI codepage : 1252
Plugins : 
    mimeTools (3.1)
    NppConverter (4.6)
    NppExport (0.4)
    PythonScript (3.0.18)

---

Probably introduced when #312 was solved.

**Comment by alankilborn** [comment]
Problem still present in 3.0.19.

@chcg - What was the point in releasing 3.0.19 without a fix for such a huge bug?

**Comment by chcg** [comment]
@alankilborn Mainly because of the many CVE fixes for python 3.12.6 .

**Comment by chcg** [comment]
@alankilborn Seems the initialisation config setting is also relevant here. With LAZY it is not happening, just with ATSTARTUP.

**Comment by alankilborn** [comment]
Seems to be solved in 3.0.20.
Great work!

**Comment by chcg** [comment]
The automatically generated part of the release notes just contains a limited amount of changes. I added this change manually.
